python/py-beancount/Portfile

```diff
@@ -24,7 +24,8 @@ homepage            http://furius.ca/beancount/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-python.versions     37 38
+python.versions     37 38 39 310
+python.default_version  38
```

Comment:
If possible, the default version should probably now be 3.10 (or the latest it will support.)